content/700-optimize/300-recording.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: 'Recording'
+metaTitle: 'Optimize: Recording'
+metaDescription: "Learn about using Optimize's recording feature."
+tocDepth: 3
+toc: true
+---
+
+<TopBlock>
+
+The Recordings feature is designed to help developers debug and isolate different sets of queries into distinct sessions, known as recordings. This targeted approach enables precise performance analysis and optimization by preventing the mixing of queries from different applications or test rounds, leading to clearer insights and more effective debugging.
+
+</TopBlock>
+
+## Managing a recording session
+
+You can manually start and stop recording sessions via the [Optimize dashboard](https://optimize.prisma.io) by clicking the **Start Recording** and **Stop Recording** buttons.
+
+:::warning
+
+A recording will automatically stop if the 10k query limit is reached or if the Prisma schema of the app is changed.
+
+:::
+
+A recording session can be deleted when no longer needed.
+
+## Identifying a recording session
+
+You can rename and tag your recording sessions for easy identification and context. Click on the generated default name of the recording session and type the desired name.
+
+## Data captured in a recording session
+
+Each recording session captures insights about queries executed in your app, including:
+- All executed queries during the session.
+- **Raw Query**: The raw query generated and sent to the database by Prisma ORM.
+- **Count**: The number of times a query pattern was executed.
+- [Query performance metrics](/optimize/performance-metrics).
+- Errors encountered during query execution.
```

Comment:
the differences/inconsistencies in formatting and punctuation is a bit irritating here, could that be consolidated (e.g. start _all_ bullets with a bold keyword and a colon).